fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -1061,11 +1073,15 @@ ACTOR Future<bool> checkExclusion(Database db,
 		return false;
 	}
 
-	double finalFreeRatio = 1 - (totalKvStoreUsedBytes / (totalKvStoreUsedBytesNonExcluded + totalKvStoreFreeBytes));
-	if (ssExcludedCount == ssTotalCount || finalFreeRatio <= 0.1) {
-		std::string temp = "ERROR: This exclude may cause the total free space in the cluster to drop below 10%.\n"
+	// The numerator is the total space in use by FDB that is not immediately reusable;
+	// the denominator is the total space usable by FDB (either used or unused currently)
+	double finalUnavailableRatio = (totalKvStoreUsedBytes + totalKvStoreFreeBytes - totalKvStoreAvailableBytes) /
```

Comment:
It might be worth explaining here why this math works, which is based on Available being (free + reusable) so Used + Free - Avail = Used + Free - Free - Reusable = Used - Reusable

Reply:
Done